### ISSUE TITLE
Official Flash Player projector apparently still works!

### ISSUE THREAD

**Issue by Hexstream** [pakastin/open-source-flash#142]
Surprisingly enough, I can confirm that the official Flash Player projector still [works for me](https://www.hexstream.net/#23_february_2021) on Linux.

That's especially great if you want to reference the original flash content to ensure you make an accurate port.

**Comment by parkerlreed** [comment]
The projector has a timer in it as well. There's a patch floating around.

**Comment by parkerlreed** [comment]
https://gist.github.com/KuromeSan/56d8b724c0696b54f9f81994ae3591d1

**Comment by Hexstream** [comment]
> The projector has a timer in it as well. There's a patch floating around.

I just downloaded [the official Flash Player projector](https://www.adobe.com/support/flashplayer/debug_downloads.html) and now I can apparently run any flash content, no patching needed. You just launch the app and input the SWF file URL, [per my post](https://www.hexstream.net/#23_february_2021).

**Comment by JPcode05** [comment]
> > The projector has a timer in it as well. There's a patch floating around.
> 
> I just downloaded [the official Flash Player projector](https://www.adobe.com/support/flashplayer/debug_downloads.html) and now I can apparently run any flash content, no patching needed. You just launch the app and input the SWF file URL, [per my post](https://www.hexstream.net/#23_february_2021).

By the way, the projector is already included in [BlueMaxima's Flashpoint](https://bluemaxima.org/flashpoint/). Hope they use the patch...

**Comment by parkerlreed** [comment]
Yes. Flashpoint either holds back the version or uses patched where needed (Best I recall from lurking on the Discord)
